charts/actions-runner-controller/docs/UPGRADING.md

```diff
@@ -29,6 +29,15 @@ curl -L https://github.com/actions/actions-runner-controller/releases/download/a
 kubectl replace -f crds/
 ```
 
+For CHART_VERSION=0.22.0+, deploy Prometheus CRDs
```

Comment:
I believe these Prometheus-operator-related CRDs are required for everything related to `serviceMonitor`. It might not be specific to chart 0.22.0+ or actionsmetrics-server.

Thats said, it would great to rephrase this a bit, like:

```suggestion
In case you're going to create prometheus-operator `ServiceMonitor` resources via the chart, you'd need to deploy prometheus-operator-related CRDs as well.

For example, you can use the `appscode/prometheus-operator-crds` to deploy those CRDs with Helm:
```